config/dexs.json

```diff
@@ -469,6 +469,12 @@
     "0x677Fa29FFe6c8f03D6bbE789090Dceb498b7aaA4",
     "0x1a4E99aB56BBac95810C0A957F173054f6FA8fDc"
   ],
+  "ink": [
+    "0x8295805320853d6B28778fC8f5199327e62e3d87",
```

Comment:
Quick question: Is is a standard that we put FeeCollector contract address in `dexs.json`? Could you elaborate?

Reply:
@mirooon 

Yes, the FeeCollector is a periphery (=external) contract that the diamond calls (like a DEX) to collect fees. Since the diamond can only call external contracts that are whitelisted, it is required that each of those contracts must be whitelisted first.  In our global.json you find the list of contracts we currently automatically whitelist when we deploy a new network:
```
  "autoWhitelistPeripheryContracts": [
    "FeeCollector",
    "GasZipPeriphery",
    "TokenWrapper",
    "LiFiDEXAggregator"
  ],

```

Reply:
ok, thanks!That makes sense. what do you think about changing file name from `dexs.json` to smth like: `whitelisted.json`? It might better reflect the purpose of the file. But it's another topic not for this PR as it will require more changes.

Reply:
Yes, that is a good suggestion. I will add it to the upcoming revision of the whole whitelisting process and format...Thanks @mirooon